src/Psalm/Fork/Pool.php

```diff
@@ -253,15 +253,29 @@ function ($data) {
      */
     public function wait()
     {
+        global $argv;
 
         // Read all the streams from child processes into an array.
         $content = $this->readResultsFromChildren();
 
+        $command = $argv[0];
+
         // Wait for all children to return
         foreach ($this->child_pid_list as $child_pid) {
-            posix_kill($child_pid, SIGALRM);
-            if (pcntl_waitpid($child_pid, $status) < 0) {
-                error_log(posix_strerror(posix_get_last_error()));
+            /** @psalm-suppress ForbiddenCode */
+            $process_lookup = @shell_exec(
```

Comment:
Can this use `--pid`? Also, should check if windows once you're done prototyping.

```
       --pid pidlist
              Select by process ID.  Identical to -p and p.
```

Reply:
Ah thanks!

Re Windows: I [commented out](https://github.com/vimeo/psalm/pull/1006/commits/91376b8cdc665d033baa21bb76f65007692db8c9#diff-495e0ae9fc6e4853664145d5a9e1985bR915) the bit in `getCpuCount` where we get the correct count of windows processes - given Windows doesn't support `pcntl`, this area should be all *nix-only.